src/install/dotnet/installation/aws-elastic.mdx

```diff
@@ -1,33 +1,19 @@
 ---
-title: Install the .NET agent on AWS Elastic Beanstalk
-tags:
-  - Agents
-  - NET agent
-  - Other installation
-metaDescription: How to install the APM .NET agent on AWS Elastic Beanstalk.
-redirects:
-  - /docs/agents/net-agent/install-guides/install-net-agent-on-aws-elastic-beanstalk
-freshnessValidatedDate: never
+componentType: default
+optionType: deployment
+headingText: Install the agent
+descriptionText: ""  
 ---
 
-Amazon Web Services (AWS) [Elastic Beanstalk](https://aws.amazon.com/elasticbeanstalk) is a dynamic service that allows easy deployment and scalability for your applications. Follow these instructions to deploy the .NET agent to the instances launched with your .NET AWS Elastic Beanstalk applications. You may also want to install the New Relic Infrastructure agent on your Elastic Beanstalk instances; refer to [Configure the infrastructure agent on AWS Elastic Beanstalk](/docs/infrastructure/install-infrastructure-agent/config-management-tools/configure-infrastructure-agent-aws-elastic-beanstalk/) for more details.
+<Callout variant="tip">
+If you haven't already, you may want to [install the New Relic Infrastructure agent on your Elastic Beanstalk instances](/docs/infrastructure/install-infrastructure-agent/config-management-tools/configure-infrastructure-agent-aws-elastic-beanstalk/) for more details.
```

Comment:
"may" here is a bit passive. It's fine if you can't find a better way to phrase it (I can't), but there might be a more direct and opinionated version of this